etc/fiddle-core.api.md

```diff
@@ -89,6 +89,11 @@ export class ElectronVersions extends BaseVersions {
     get versions(): SemVer[];
 }
 
+// @public (undocumented)
+export interface ElectronVersionsCreateOptions {
+    initialVersions?: unknown;

```

Comment:
Perhaps `initialVersions?: Array<Record<"version", string>>` would be better?

Reply:
That would probably be better, yea, but other places (like line 17 in this file) use `unknown` for `versions`, so I've remained consistent across all usages. We can tighten up the typings there in a follow-up PR.